src/Microsoft.ApplicationInsights.AspNet/Tracing/Aspnet5EventSource.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace Microsoft.ApplicationInsights.AspNet.Tracing
+{
+    using System;
+    using System.Diagnostics.Tracing;
+
+    [EventSource(Name = "Microsoft-ApplicationInsights-AspNet5")]
+    public class AspNet5EventSource : EventSource
+    {
+        public AspNet5EventSource()
+        {
+        }
+
+        [Event(1, Message = "Telemetry Initializer will not collect data when HttpContext is null.", Level = EventLevel.Verbose)]
+        public void VerboseTelemetryInitializerNotEnabledOnHttpContextNull()
```

Comment:
Why do we want to include level (error, verbose) and keyword (UserActionable) in the names of logging methods?